src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -23,15 +30,28 @@ export const portfolio = createSlice({
       moduleLogger.info('clearing portfolio')
       return initialState
     },
-    setWalletId: (state, { payload }: { payload: WalletId | undefined }) => {
+    setWalletMeta: (state, { payload }: { payload: WalletMetaPayload }) => {
```

Comment:
we needed both wallet id and name to be able to fire the track event, also one less render to set/unset these together